src/Workspaces/Core/Portable/Formatting/Rules/BaseIndentationFormattingRule.cs

```diff
@@ -187,7 +187,7 @@ private static TextSpan GetSpanFromTokens(TextSpan span, SyntaxToken token1, Syn
             // current token is inside of the given span, get next token's start position.
             if (token2.Span.End <= span.End)
             {
-                token2 = token2.GetNextToken();
+                token2 = token2.GetNextToken(includeZeroWidth: true);
```

Comment:
@dpoeschl  @heejaechang We dont do it for GetPreviousToken() because the auto generated C# buffer content always have a well formed C# syntax in the beginning of the file. Theoretically speaking, there can be a crash through the GetPreviousToken calculation as well but practically it is impossible(No repro case).